addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3698,6 +3698,9 @@ export class OdooEditor extends EventTarget {
                             );
                         }
                         tab.remove();
+                        if (!sel.isCollapsed) {
+                            this._updateToolbar(true);
+                        }
```

Comment:
@visp-odoo this part which you are trying to handle looks fixed, so remove this ! Thanks

Reply:
Done :+1: